docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -41,8 +41,8 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
   * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
   * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
-  * `Python 3.6 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.10 are recommended. 
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
```

Comment:
master has been changed to 3.10 according to this PR: https://github.com/openvinotoolkit/openvino/pull/13243

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/13298/commits/5ff36d9c35ef6344b497ec17dcedc56e264a0f91